src/core_plugins/kbn_vislib_vis_types/public/area.js

```diff
@@ -1,19 +1,25 @@
+import VisVisTypeProvider from 'ui/vis/vis_type';
```

Comment:
Unfortunately, we need to import VisType, because `CATEGORIES` is assigned as a property to that class object/function, and an extended class won't inherit it.

I actually question whether inheritance is appropriate in this system. I feel like we should take a `has-a` approach instead of `is-a`. We should configure VisType directly with whatever functionality it needs, instead of subclassing it.